tests/behaviour/block_data_test.py

```diff
@@ -26,6 +27,8 @@ async def test_starknet():
     )
 
     starknet = await StarknetState.empty()
+    starknet.state.block_info = BlockInfo(0xDEADBEEF, 0xCAFEBABE)
```

Comment:
I'd prefer to give explicit names to parameters (`block_number=0xdeadbeef, block_timestamp=…` )

Reply:
(it just wasn't very clear at a first glance which was which)

Reply:
fixed